source/languages/en/riak/ops/advanced/backends/multi.md

```diff
@@ -107,8 +107,8 @@ to set LevelDB's `bloomfilter` setting to `off` and the Bitcask
 backend's `io_mode` setting to `nif`, you would do that as follows:
 
 ```riakconf
-multi_backend.leveldb.bloomfilter = off
-multi_backend.bitcask.io_mode = nif
+multi_backend.my_backend.leveldb.bloomfilter = off
+multi_backend.my_backend.bitcask.io_mode = nif
```

Comment:
I agree with the spirit of this edit, but we would want to have two different names for the backends, since the name is what refers to the specific backend.
like:

```
multi_backend.my_leveldb_backend.leveldb.bloomfilter = off
multi_backend.my_bitcask_backend.bitcask.io_mode = nif
```